package.json

```diff
@@ -6,123 +6,73 @@
-    "examples-css-checkout:build": "node-sass examples/checkout/styles/styles.scss > .dist-test/examples/checkout/index.css",
-    "examples:copy": "cp -r examples .dist-test/examples",
-    "------ Script used print out a nice start message once start is ready to go": null,
-    "start:wait-print-msg": "wait-on file:.dist-test/sdk-testing.js file:.dist-test/tests.js tcp:4200 && echo \"\n\n⚡️⚡️⚡️  Good to go at http://localhost:4200 ⚡️⚡️⚡️\""
+    "build:dev": "rollup -c --environment BABEL_ENV:node",
+    "build:prod": "rollup -c --environment BABEL_ENV:node,BUILD_MODE:production",
+    "start": "npm run clean && npm run setup-paths && concurrently 'npm run test:web-deps' 'npm run test:livereload-server' 'npm run test:watch' 'http-server -p 4200 .tmp/test' 'npm run -s print-start-message'",
+    "clean": "rimraf .tmp types.js",
+    "setup-paths": "mkdir -p .tmp/test",
+    "test": "npm run clean && npm run setup-paths && npm run test:node-deps && npm run -s mocha",
+    "test:watch": "cross-env BABEL_ENV='browser' node -- scripts-graphql/test-build-watch.js --with-dependency-tracking .tmp/test/tests.js",
+    "test:web-deps": "concurrently 'npm run test:copy-web-deps' 'npm run schema:transform-to-types'",
+    "test:node-deps": "npm run schema:transform-to-types && cross-env BABEL_ENV='node' node -- scripts-graphql/test-build-node.js --with-dependency-tracking .tmp/test/node-tests.js",
+    "test:copy-web-deps": "cp test-graphql/index.html node_modules/mocha/mocha.* node_modules/jquery/dist/jquery.slim.min.js .tmp/test/ && cp node_modules/expect.js/index.js .tmp/test/expect.js",
+    "test:livereload-server": "node scripts-graphql/livereload.js",
+    "mocha": "mocha -u tdd $(npm run -s mocha:reporter-args) .tmp/test/node-tests.js",
+    "mocha:reporter-args": "[ -n \"${CI}\" ] && echo --reporter xunit --reporter-options output=$CIRCLE_TEST_REPORTS/junit/mocha.xml",
+    "lint": "eslint --max-warnings 0 -c .eslintrc.json $(npm run -s lint:reporter-args) src-graphql/ test-graphql/",
+    "lint:reporter-args": "[ -n \"${CI}\" ] && echo -o $CIRCLE_TEST_REPORTS/junit/eslint.xml -f junit",
+    "print-start-message": "wait-on file:.tmp/test/index.html file:.tmp/test/tests.js tcp:35729 tcp:4200 file:types.js && echo \"\n\n⚡️⚡️⚡️  Good to go at http://localhost:4200 ⚡️⚡️⚡️\"",
+    "schema:refresh": "rimraf types.js && npm run schema:fetch && npm run schema:transform-to-types",
+    "schema:fetch": "graphql-js-schema-fetch --url 'https://graphql.myshopify.com/api/graphql' --header 'Authorization: Basic MzUxYzEyMjAxN2QwZjJhOTU3ZDMyYWU3MjhhZDc0OWM=' | jq '.' > schema.json",
+    "schema:transform-to-types": "graphql-js-schema --schema-file schema.json --outdir . --schema-bundle-name 'Types' --bundle-only"
   },
   "author": "Shopify Inc.",
   "license": "MIT",
-  "dependencies": {
-    "whatwg-fetch": "0.10.1",
-    "node-fetch": "1.3.3",
-    "core-js": "2.4.1",
-    "Base64": "0.3.0"
-  },
+  "dependencies": {},
```

Comment:
For now, I don't need any, because rollup will bundle them in where used. As real work actually starts happening in here, it may change.